osdk/README.md

```diff
@@ -47,6 +44,17 @@ cargo install --force cargo-osdk
 
 Here we provide a simple demo to demonstrate how to create and run a simple kernel with `cargo-osdk`.
 
+Suppose you are on a x86_64 ubuntu machine, to run a kernel with QEMU, the following tools should be installed:
+- qemu-system-x86_64
+- grub-mkrescue
+- ovmf 
+- xorriso
+
+If these tools are missing, they can be installed by
+```bash
+apt install grub2-common qemu-system-x86 ovmf xorriso
```

Comment:
It seems the `README.md` of osdk does not rebase to the newest main branch? The `README.md` file content have been updated. You can copy the lastest content from main branch.